website/sidebars.js

```diff
@@ -15,12 +15,14 @@ module.exports = {
     {
       type: 'category',
       label: 'Getting Started',
+      go_to: 'configuration',
```

Comment:
I changed this layout to `go_to` to `link: {type: 'doc', id: 'presets'}` to get ready for the next commit that will change from comparing to label to comparing to the id instead.